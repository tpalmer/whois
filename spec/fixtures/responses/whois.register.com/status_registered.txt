
The data in Register.com's WHOIS database is provided to you by
Register.com for information purposes only, that is, to assist you in
obtaining information about or related to a domain name registration
record.  Register.com makes this information available "as is," and
does not guarantee its accuracy.  By submitting a WHOIS query, you
agree that you will use this data only for lawful purposes and that,
under no circumstances will you use this data to: (1) allow, enable,
or otherwise support the transmission of mass unsolicited, commercial
advertising or solicitations via direct mail, electronic mail, or by
telephone; or (2) enable high volume, automated, electronic processes
that apply to Register.com (or its systems).  The compilation,
repackaging, dissemination or other use of this data is expressly
prohibited without the prior written consent of Register.com.
Register.com reserves the right to modify these terms at any time.
By submitting this query, you agree to abide by these terms.


   Registrant:
      Register.Com, Inc.
      Domain Registrar
      575 8th Avenue
      New York, NY 10018
      US
      Phone: +1.9027492701
      Email: domainregistrar@register.com



   Registrar Name....: Register.com
   Registrar Whois...: whois.register.com
   Registrar Homepage: www.register.com

   Domain Name: register.com
      Created on..............: 1994-11-01
      Expires on..............: 2019-08-04

   Administrative Contact:
      Register.Com, Inc.
      Domain Registrar
      575 8th Avenue
      New York, NY 10018
      US
      Phone: +1.9027492701
      Email: domainregistrar@register.com


   Technical  Contact:
      Register.Com, Inc.
      Domain Registrar
      575 8th Avenue
      New York, NY 10018
      US
      Phone: +1.9027492701
      Email: domainregistrar@register.com


   DNS Servers:
      ns2.register.com
      ns3.register.com
      ns4.register.com
      ns5.register.com
      ns6.register.com
      ns1.register.com



Visit AboutUs.org for more information about register.com

<A HREF="http://www.aboutus.org/register.com">AboutUs: register.com</A>

Register your domain name at http://www.register.com

