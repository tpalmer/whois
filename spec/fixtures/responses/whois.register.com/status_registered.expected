#status
  should: %s raise_error(Whois::PropertyNotSupported)

#available?
  should: %s == false

#registered?
  should: %s == true


#created_on
  should: %s CLASS(time)
  should: %s == Time.parse("1994-11-01")

#updated_on
  should: %s raise_error(Whois::PropertyNotSupported)

#expires_on
  should: %s CLASS(time)
  should: %s == Time.parse("2019-08-04")


#registrar
  should: %s CLASS(registrar)
  should: %s.id           == nil
  should: %s.name         == "Register.com"
  should: %s.url          == "http://www.register.com/"


#registrant_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type         == Whois::Record::Contact::TYPE_REGISTRANT
  should: %s[0].name         == "Domain Registrar"
  should: %s[0].organization == "Register.Com, Inc."
  should: %s[0].address      == "575 8th Avenue"
  should: %s[0].city         == "New York"
  should: %s[0].zip          == "10018"
  should: %s[0].state        == "NY"
  should: %s[0].country_code == "US"
  should: %s[0].phone        == "+1.9027492701"
  should: %s[0].fax          == nil
  should: %s[0].email        == "domainregistrar@register.com"

#admin_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type         == Whois::Record::Contact::TYPE_ADMIN
  should: %s[0].name         == "Domain Registrar"
  should: %s[0].organization == "Register.Com, Inc."
  should: %s[0].address      == "575 8th Avenue"
  should: %s[0].city         == "New York"
  should: %s[0].zip          == "10018"
  should: %s[0].state        == "NY"
  should: %s[0].country_code == "US"
  should: %s[0].phone        == "+1.9027492701"
  should: %s[0].fax          == nil
  should: %s[0].email        == "domainregistrar@register.com"

#technical_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type         == Whois::Record::Contact::TYPE_TECHNICAL
  should: %s[0].name         == "Domain Registrar"
  should: %s[0].organization == "Register.Com, Inc."
  should: %s[0].address      == "575 8th Avenue"
  should: %s[0].city         == "New York"
  should: %s[0].zip          == "10018"
  should: %s[0].state        == "NY"
  should: %s[0].country_code == "US"
  should: %s[0].phone        == "+1.9027492701"
  should: %s[0].fax          == nil
  should: %s[0].email        == "domainregistrar@register.com"


#nameservers
  should: %s CLASS(array)
  should: %s SIZE(6)
  should: %s[0] CLASS(nameserver)
  should: %s[0].name == "ns2.register.com"
  should: %s[1] CLASS(nameserver)
  should: %s[1].name == "ns3.register.com"
  should: %s[2] CLASS(nameserver)
  should: %s[2].name == "ns4.register.com"
  should: %s[3] CLASS(nameserver)
  should: %s[3].name == "ns5.register.com"
  should: %s[4] CLASS(nameserver)
  should: %s[4].name == "ns6.register.com"
  should: %s[5] CLASS(nameserver)
  should: %s[5].name == "ns1.register.com"
